### ISSUE TITLE
Cannot unblock many sites

### ISSUE THREAD

**Issue by Sapiosenses** [andryou/scriptsafe#199]
I have had frequent problems with not being able to get pages to load despite trusting everything possible I can trust on these problematic pages.

Also many times the list of blocked URLs/domains doesn't show up initially. All that shows up is the primary root domain, I have to close/reopen the Scriptsafe panel in order to see them. (Which also, annoyingly, seems to force a page reload each time I do that)

Latest example:

https://my.pair.com/

(I'm using Vivaldi - 1.8 latest on Windows 7- and while I realize you've had some issues with it, all I can find in the buglist are issues with auto page reloading on scriptsafe changes.)

**Comment by 7eggert** [comment]
Some scripts will load additional scripts. They are not part of the original website - you do need to reload the page in order to activate the unblocked scripts and to see the additional scripts. This cannot be avoided for inline scripts and is a hassle to avoid on the minority of scripts that are supposed to run asynchronously or deferred.

Not having a pair account, I couldn't reproduce the problem. Please retry and close this bug if you can log in. Also try to verify weather a cookie-blocking plugin might be the cause of not being able to log in.

(I am not one of the developers).
